packages/react-components/react-infobutton/src/components/InfoButton/useInfoButtonStyles.styles.ts

```diff
@@ -27,6 +27,7 @@ const useButtonStyles = makeStyles({
 
     backgroundColor: tokens.colorTransparentBackground,
     color: tokens.colorNeutralForeground2,
+    cursor: 'pointer',
```

Comment:
To differentiate between the InfoButton and InfoTip, we're adding the cursor pointer to make it easier to tell InfoButton has a button.